python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -79,6 +79,16 @@
   private static final Set<String> BUILTINS_TO_DISAMBIGUATE = new HashSet<>(
     Arrays.asList(INT, FLOAT, COMPLEX, STR, BuiltinTypes.SET, DICT, LIST, TUPLE, NONE_TYPE, BOOL, "type", "super", "frozenset", "memoryview"));
 
+  private static final Map<String, String> MODULES_TO_DISAMBIGUATE = new HashMap<>();
+  static {
+    MODULES_TO_DISAMBIGUATE.put("ConfigParser", "2@ConfigParser");
+    MODULES_TO_DISAMBIGUATE.put("Queue", "2@Queue");
+    MODULES_TO_DISAMBIGUATE.put("SocketServer", "2@SocketServer");
+    MODULES_TO_DISAMBIGUATE.put("configparser", "configparser");
+    MODULES_TO_DISAMBIGUATE.put("queue", "queue");
+    MODULES_TO_DISAMBIGUATE.put("socketserver", "socketserver");
```

Comment:
I guess those aren't even needed since we'll default to the `moduleName` anyway when the entry is not found, right?
Also we probably could have a short comment explaining those modules differ in Python 2 and Python 3 by capitalization only.